runtime/coreclr-bridge.m

```diff
@@ -492,7 +472,7 @@
 MonoClass *
 mono_get_string_class ()
```

Comment:
inconsistent (MonoClass* vs MonoType*) with the above ones

Reply:
That's because `mono_get_string_class` is used differently (changing the call sites to take a MonoType is more complex, and doesn't really make things better, which it did for the other two functions).